bucket/vagrant-manager.json

```diff
@@ -0,0 +1,21 @@
+ {
+    "version": "2.3.2",
+    "description": "Status bar menu for managing all Vagrant machines from one central location",
+    "homepage": "https://github.com/absalomedia/vagrant-manager",
+    "license": "MIT",
+    "url": "https://github.com/absalomedia/vagrant-manager/releases/download/2.3.2/vagrant-manager.exe",
+    "bin": "vagrant-manager.exe",
+    "shortcuts": [
+        [
+            "vagrant-manager.exe",
+            "Vagrant Manager"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/absalomedia/vagrant-manager",
+        "regex": "/tag/([-\\w.]+)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe"
+    }
+}

```

Comment:
```suggestion
    "suggest": {
        "Vagrant": "vagrant"
    },
    "url": "https://github.com/absalomedia/vagrant-manager/releases/download/2.3.2/vagrant-manager.exe#/dl.7z",
    "hash": "c6a99b2e5b18f384ac5b25abc539c319efe9b3cd472c9b73a01130826b1f6240",
    "architecture": {
        "64bit": {
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
            }
        },
        "32bit": {
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\""
            }
        }
    },
    "post_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse",
    "shortcuts": [
        [
            "Vagrant Manager.exe",
            "Vagrant Manager"
        ]
    ],
    "checkver": "github",
    "autoupdate": {
        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe#/dl.7z"
    }
```

Reply:
I downloaded the vagrant-manager-2.3.2-x64.nsis.7z package to verify it. Extracted out fine. Ran fine. Windows 10 Pro running v1809. 

The Vagrant Manager.exe file inside the NSIS 7z file did what it was supposed to do. End up in the Windows taskbar. It's not an installer, it's the final app.

Reply:
Yes I know it is not installer... And yes it is executed in taskbar, but right click do not work. It do nothing

- Sandbox 1903
- Server 2019
- Server 2012R2
- Windows 20H1
- Windows 1909

Reply:
I'll check on 1909 once the feature update goes through on my machine tonight.

Reply:
Apply this changes, since current manifest is not working.

Reply:
Applied changed to JSON based on your feedback